impl/http/src/main/java/io/serverlessworkflow/impl/executors/http/auth/requestbuilder/OAuthRequestBuilder.java

```diff
@@ -15,37 +15,50 @@
  */
 package io.serverlessworkflow.impl.executors.http.auth.requestbuilder;
 
+import static io.serverlessworkflow.impl.WorkflowUtils.concatURI;
+import static io.serverlessworkflow.impl.executors.http.SecretKeys.AUTHORITY;
+
 import io.serverlessworkflow.api.types.OAuth2AuthenticationPropertiesEndpoints;
 import io.serverlessworkflow.api.types.OAuth2ConnectAuthenticationProperties;
 import io.serverlessworkflow.impl.WorkflowApplication;
+import io.serverlessworkflow.impl.WorkflowUtils;
+import io.serverlessworkflow.impl.WorkflowValueResolver;
 import java.net.URI;
 import java.util.Map;
 
-public class OAuthRequestBuilder extends AbstractAuthRequestBuilder {
+public class OAuthRequestBuilder
```

Comment:
We may even consider this lightweight dep: https://www.pac4j.org/docs/clients/openid-connect.html (maybe as an optional library `impl-http-oidc`)

Reply:
yes, this partially implemented, I will open PR for completion